Src/node_builder.py

```
@@ -76,30 +76,23 @@ def _build_list_node(self, node_data: NodeAnnotation, parent: int | str) -> int
-            (label, param) for label, param in node_data.annotations.items()
-            if label != 'INPUT' and not isinstance(param.hint, ANode)
-        ]
-        connection_params = [
-            label for label, param in node_data.annotations.items()
-            if label != 'INPUT' and isinstance(param.hint, ANode)
-        ]
+        params = [(label, param) for label, param in node_data.annotations.items() if label != 'INPUT']
 
-        items_count = len(params) + len(connection_params)
+        items_count = len(params)
         if node_data.input: items_count += 1
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)